ia/src/process.py

```
@@ -2,8 +2,9 @@
 from models import iaParser
 from parser import ia_parser
 from utils import exit_error
+from message import *
 
-def iaProcess(argv: list[str]):
+def iaProcess(argv: list[str]) -> None:
```

Pylint (3.3.7)

[convention] Function name "iaProcess" doesn't conform to snake_case naming style

(C0103)